src/clusterfuzz/stacktraces/__init__.py

```diff
@@ -759,15 +759,29 @@ def parse(self, stacktrace: str) -> CrashInfo:
             type_filter=get_fault_description_for_android_kernel)
 
       # Generic KASan errors.
-      if self.update_state_on_match(
+      address_match = self.update_state_on_match(
```

Comment:
Nit: no need for these variables. Follow the existing style, adding comments where necessary.

Reply:
1. first and second cases do slightly different things: namely the first adds `0x` to the crash_address. Second case does not.
2. When I was following the existing style, linter kept it quite ugly with 10+ lines crammed in `if()` statement. Not easily readable.

Reply:
Re 1) Yes they are two cases so I'm suggesting that we have two ifs without these vars. 

Re 2) It's much better for overall readability to be consistent here with the rest of the code in this function/file.

Reply:
Oh I get what you mean. Thanks for clarifying

Reply:
fixed